test/integration/aws/function-url.test.js

```diff
@@ -110,4 +120,14 @@ describe('test/integration/aws/function-url.test.js', function () {
     expect(headers.get('access-control-expose-headers')).to.equal('x-foo');
     expect(headers.get('access-control-allow-credentials')).to.equal('true');
   });
+
+  it('should return streaming invoke mode Lambda URL', async () => {
```

Comment:
Is it confirmed that test passes?

Reply:
Can you guys merge this please, im tryna use it

Reply:
@medikoo It works locally. I am waiting for someone to approve running tests in CI:

```
$ AWS_ACCESS_KEY_ID=*** AWS_SECRET_ACCESS_KEY="***" npx mocha test/integration/aws/function-url.test.js 
(node:846961) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)


  test/integration/aws/function-url.test.js
2023-05-27T21:05:08.096Z ℹ serverless:test deploy test-function-2fsph87s (at /tmp/tmpdirs-serverless/a99c/11b324)
    ✔ should return valid response from Lambda URL (1008ms)
    ✔ should return valid response from Lambda URL with authorizer with valid signature (891ms)
    ✔ should return invalid response from Lambda URL with authorizer without passed signature (338ms)
    ✔ should return expected CORS headers from Lambda URL (777ms)
    ✔ should return streaming invoke mode Lambda URL (854ms)
2023-05-27T21:07:00.757Z ℹ serverless:test remove test-function-2fsph87s (at /tmp/tmpdirs-serverless/a99c/11b324)


  5 passing (2m)

```

Reply:
Integration tests are not run on PR's (only after merge's)